test/nightly/crud.test.ts

```diff
@@ -36,7 +32,7 @@ suite('Resource CRUD Operations', function (this: Mocha.Suite): void {
             const testContext = await createTestActionContext();
             testSubscription = subscriptionTreeItems[0] as SubscriptionItem;
             const context = { ...testContext, ...testSubscription.subscription };
-            locations = await LocationListStep.getLocations(context);
+            locations = (await testApi.testing.getLocations(context)).slice(0, 5); // limit to 5 locations for test speed
```

Comment:
Any problem with doing just 5 instead of the 49 we were doing?

Reply:
That's probably fine